scripts/searchSamadi.py

```diff
@@ -0,0 +1,20 @@
+import commands, sys
+
+stringToLookFor = "*v0.1.0+76X_HHAnalysis_2016-04-04.v0"
+output = commands.getoutput("search_SAMADhi.py sample -n %s"%stringToLookFor)
+# If you want specific samples in a given prod : output = commands.getoutput("search_SAMADhi.py sample -n %s | grep -v 'DYJets.*HT.*Spring.*'"%stringToLookFor)
```

Comment:
@Olivier If I remember well, you can use this to avoid having the merged DY (maybe changing the "Spring" to "Fall" for 76).  The commit of this script was just to ease Seb W work. It is not generic at all, e.g. you also need to change the "signal" string line 15. If there are demand, we can make it more generic easily.